fontforge/python.c

```diff
@@ -44,6 +44,17 @@
 #define ENDPYGETSTR()
 #endif
 
+#if PY_VERSION_HEX > 0x03000000
+#define PYCMPF(a) NULL
+#define PYRCMPF(b) b
+#elif PY_VERSION_HEX > 0x02020000
```

Comment:
Support for "Rich comparison" was apparently added in this release and support for normal comparison dropped in 3, hence the three combinations.

Reply:
Just a note, but I believe the build system enforces Python >= 2.7